source/documentation/monitoring_apps/logs.md

```diff
@@ -43,16 +43,25 @@ You must set up [logstash](https://www.elastic.co/products/logstash) to process
 1. Go to the __Logstash Filters__ page, and replace the code there with the following logstash filter code:
 
     ```
+    # updated 2020-07-01
     filter {
         grok {
             # attempt to parse syslog lines
-            match => { "message" => "%{SYSLOG5424PRI}%{NONNEGINT:syslog_ver} +(?:%{TIMESTAMP_ISO8601:syslog_timestamp}|-) +(?:%{HOSTNAME:syslog_host}|-) +(?:%{NOTSPACE:syslog_app}|-) +(?:%{NOTSPACE:syslog_proc}|-) +(?:%{WORD:syslog_msgid}|-) +(?:%{SYSLOG5424SD:syslog_sd}|-|) +%{GREEDYDATA:syslog_msg}" }
+            match => { "message" => "(%{NONNEGINT:message_length} )?%{SYSLOG5424PRI}%{NONNEGINT:syslog_ver} (?:%{TIMESTAMP_ISO8601:syslog_timestamp}|-) +%{DATA:syslog_host} +%{UUID:cf_app_guid} +\[%{DATA:syslog_proc}\] +- +(\[tags@%{NONNEGINT} +%{DATA:cf_tags}\])? +%{GREEDYDATA:syslog_msg}" }
```

Comment:
probably does't make much difference ... but these tags@ [numbers are actually constant]( https://github.com/cloudfoundry/loggregator-agent-release/blob/800d8ffb1c1ebb2465cf54f6e5db18b69be7ed33/src/pkg/egress/syslog/rfc5424.go#L32)

Reply:
_Enterprise_ :D
I'm not using them for anything so I think non-negative integer is probably fine